### PR TITLE
refresh GHA runners for UBI8 tests

### PR DESCRIPTION
**Pull request by jmtd**
This is intended to be the UBI8 equivalent of #418 

It rolls up a fix for https://issues.redhat.com/browse/OPENJDK-2438 (commit references the UBI9 variant of that bug, OPENJDK-2414)

### CODE REVIEW

**Comment by jmtd** [comment]
@sefroberg most of this PR is stuff specific to running the Behave tests on GitHub actions, but there are some label changes from https://issues.redhat.com/browse/OPENJDK-2438 in there too, FYI. They're included because the tests for them were included already but failing.

File: tests/features/java/java_s2i.feature [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# temporarily marking 'ignore' so these tests are skipped on GHA
+@ignore
```
Comment (jerboaa) [inline tests/features/java/java_s2i.feature]: Please file an issue to rectify this `@ignore`. Thanks!

File: tests/features/java/openjdk_s2i.feature [hunk 2]
```diff
@@ -5,6 +5,7 @@
 Feature: Openshift OpenJDK-only S2I tests
 
   @ubi8/openjdk-8
+  @ignore
```
Comment (jerboaa) [inline tests/features/java/openjdk_s2i.feature:10]: Same. Please file an issue and reference it here.

**Review by jerboaa — COMMENTED** [review]
Looks mostly OK. Please reference issues with ignored tests. Thanks!

File: tests/features/java/openjdk_s2i.feature (hunk 2 above)
Comment (jmtd) [reply tests/features/java/openjdk_s2i.feature:10]: Good idea. I've filed https://issues.redhat.com/browse/OPENJDK-2603 (for the ubi8 variant of this probleM) and I'll push a commit to add a reference to this issue as a comment in these twe locations.

**Comment by jmtd** [comment]
The current round of test-run failures above are all caused by transient GitHub issues (error 500 when cloning various repositories). I'm kicking off re-runs.

**Review by jerboaa — APPROVED** [review]

**Pull request merged**